packages/x-components/src/x-modules/url/components/url-handler.vue

```diff
@@ -34,6 +36,8 @@
     mixins: [xComponentMixin(urlXModule)]
   })
   export default class UrlHandler extends Vue {
+    @Inject()
```

Comment:
I would try to avoid adding the snippet here, but if we have to add it, let's make it optional.

Reply:
To properly set an injection to optional, you have to specify `{ default: undefined }` as the options. I know, weird. try removing the provide of the `x-installer` to see this.